src/bin/dual-iir.rs

```diff
@@ -194,6 +194,7 @@ mod app {
         dacs: (Dac0Output, Dac1Output),
         iir_state: [[iir::Vec5<f32>; IIR_CASCADE_LENGTH]; 2],
         generator: FrameGenerator,
+        temp_sensor: stabilizer::hardware::temp_sensor::CpuTempSensor,
```

Comment:
```suggestion
        cpu_temp: stabilizer::hardware::temp_sensor::CpuTempSensor,
```

as it's not the only temperature sensor on Stabilizer.